corehq/apps/app_execution/views.py

```diff
@@ -117,6 +118,41 @@ def _get_context(request, title, url, add_parent=False, **kwargs):
     return {**context, **kwargs}
 
 
+@require_superuser_or_contractor
+@use_bootstrap5
+def workflow_log_list(request, pk):
+    logs = AppExecutionLog.objects.filter(workflow_id=pk).order_by("-started").all()
+    context = _get_context(
```

Comment:
I don't think I paid much attention to this util when reviewing the previous PR.  This is something I've long felt is a frustration with HQ code - that you essentially need to use a class-based view that inherits from like 8 others in order to have the context necessary to render the navbar, sidebar, and the like.  I'd love to see a replicable pattern for avoiding that.  An idea I had a while ago was to use a `SiteContext` object or some such that is inherited in the same way, but only deals with this context.  I could also see a functional version that chains to parent contexts.  Dunno, but I wanted to voice my support for what you're doing here.

Reply:
👍🏻 I feel the same and have to dig through a lot of code to figure out what goes in here and how it works.